packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -44,28 +46,30 @@ async function runPipelineLoadStep(
     if (!step.load) {
       continue;
     }
-    const result = await step.load({
-      fileExt: srcExt,
-      filePath: srcPath,
-      isDev,
-      isHmrEnabled,
-      // @ts-ignore: internal API only
-      log: (msg, data: any = {}) => {
-        messageBus.emit(msg, {
-          ...data,
-          id: step.name,
-          msg: data.msg && `${data.msg} [${path.relative(process.cwd(), srcPath)}]`,
-        });
-      },
-    });
-    validatePluginLoadResult(step, result);
-    if (typeof result === 'string') {
-      const mainOutputExt = step.resolve.output[0];
-      return {[mainOutputExt]: result};
-    } else if (result && typeof result === 'object') {
-      return result;
-    } else {
-      continue;
+    logger.debug(`[${step.name}] running load()`);
+    try {
+      const result = await step.load({
+        fileExt: srcExt,
+        filePath: srcPath,
+        isDev,
+        isHmrEnabled,
+        // @ts-ignore: internal API only
+        log: (msg, data: any = {}) => {
+          if (data && data.msg)
+            logger.info(`[${step.name}] ${data.msg} [${path.relative(process.cwd(), srcPath)}]`);
+        },
+      });
+      validatePluginLoadResult(step, result);
+      if (typeof result === 'string') {
+        const mainOutputExt = step.resolve.output[0];
+        return {[mainOutputExt]: result};
+      } else if (result && typeof result === 'object') {
+        return result;
+      } else {
+        continue;
+      }
+    } catch (err) {
```

Comment:
Change I’ve been thinking about: let plugins throw errors, and we’ll show the result to the user. I personally like that approach better than logging (though they can log, too)